### PR TITLE
refactor(rxScreenshotSvc): FRMW-395-convert to Quark Service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-395

- [x] Dev LGTM
- [ ] Dev LGTM
- [x] Design LGTM

---
Refactor `rxFeedback.rxScreenshotSvc` to Quark Service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `rxScreenshotSvc` Quark Service

### Demo Page
screen shot 2016-01-08 at 11 50 48 am


### API Docs
screen shot 2016-01-08 at 11 51 12 am

**Comment by 100stacks** [comment]
**TODO:** Add `{@link quarks.service:rxScreenshotSvc rxScreenshotSvc}` to `quarks.module.js`.

File: src/quarks/rxScreenshotSvc/scripts/rxScreenshotSvc.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+angular.module('encore.ui.quarks')
 /**
  * @ngdoc service
- * @name rxFeedback.service:rxScreenshotSvc
+ * @name quarks.service:rxScreenshotSvc
  * @description
  * Captures a screenshot for `rxFeedback` submission form.
  *
- * **NOTE:** Requires `html2canvas` which Encore Framework provides by default.
+ * @requires html2canvas
```
Comment (CITguy) [inline src/quarks/rxScreenshotSvc/scripts/rxScreenshotSvc.js]: Please remove this `@requires` ngdoc/jsdoc is trying to link it to internal documentation, but it doesn't lead anywhere

**Comment by 100stacks** [comment]
@CITguy, updated API Docs in https://github.com/rackerlabs/encore-ui/pull/1487#issuecomment-170073540 to reflect documentation changes. :+1:

**Comment by glynnis** [comment]
Design LGTM

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**